samples/bluetooth/central_hr_coded/sysbuild/ipc_radio/prj.conf

```diff
@@ -1,21 +1,16 @@
-CONFIG_IPC_SERVICE=y
-CONFIG_MBOX=y
-
 CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
```

Comment:
The specific configurations of samples ie. child_image/hci_ipc/prj.conf were moved to sysbuild/ipc_radio/prj.conf and adjusted with default values of ipc_radio application (from applications/ipc_radio/overlay-bt_hci_ipc.conf).
There is an issue with checking whenever the stack amount is adequete, because of lack of CI support for sysbuild.
The hci_ipc and ipc_radio should behave the same and consume similar amount of stack, but now there is no automatic way to check if it's ok to use the reduced amount of stack.

Reply:
Shall we plan such optimization in future?

Reply:
I think it is a good idea to plan a task to use stack analyzer on various configurations to estimate 'typical' stack usage for ipc_radio.